engine/remote-components/src/main/java/org/datacleaner/components/remote/RemoteTransformer.java

```diff
@@ -100,55 +105,28 @@ public void initClient() throws RemoteComponentException {
     public void closeClient() {
         logger.debug("closing '{}' @{}", componentDisplayName, this.hashCode());
         client = null;
-        // TODO: client now misses a "close" method (although Jersey client has a "destroy" method).
+    }
+
+    @Validate
+    public void validate() throws Exception {
+        CreateInput createInput = new CreateInput();
+        createInput.configuration = getConfiguration(getUsedInputColumns());
+        try {
+            cachedOutputColumns.getCachedValue(createInput);
+        } catch(RESTClientException e) {
+            if(e.getCode() == 422) {
```

Comment:
On the server side, we implemented returning 422 in case a validation fails in the transformer. I've recently read some suggestions in REST API design articles for cases when the request and also the payload is syntactically correct (if not, 400 is the candidate), but its content is not processable.

> The 422 (Unprocessable Entity) status code means the server understands the content type of the request entity (hence a 415(Unsupported Media Type) status code is inappropriate), and the syntax of the request entity is correct (thus a 400 (Bad Request) status code is inappropriate) but was unable to process the contained instructions.